run_with_lambda.py

```
@@ -0,0 +1,301 @@
+"""
+
+import asyncio
+import json
+import time
+from datetime import datetime
+from pathlib import Path
+import sys
+import logging
+import boto3
+from botocore.exceptions import ClientError
+from typing import Dict, Any
```

Ruff (0.11.9)

`typing.Dict` is deprecated, use `dict` instead

(UP035)